### ISSUE TITLE
mephisto check should output to stderr if it fails

### ISSUE THREAD

**Issue by mjkanji** [facebookresearch/Mephisto#916]
The `mephisto check` command currently uses a print statement and `click.echo()` when it fails. This results in silent errors which may not be picked up by a CI/CD tool.

For example, in [this run](https://github.com/facebookresearch/Mephisto/actions/runs/3212261950/jobs/5250974423) of the Docker Testing Matrix job, the 'Check that Mephisto was installed correctly' stage passes, but if you look at the details, `mephisto check` actually fails:

> Something went wrong.
> The provided Mephisto data directory /mephisto/data as set in /root/.mephisto/config.yml is not a directory! Please locate your Mephisto data directory and update /root/.mephisto/config.yml to point to it.

**Comment by JackUrb** [comment]
Hi @mjkanji - this is a good point, much of Mephisto hasn't been built with this particular setting in mind, but it would definitely be an improvement for us to do. I think it'd be appropriate at least here (and eventually in other scripts utilities) to switch to `print(<>, file=sys.stderr)`. Happy to review any solution you'd want to put up though!

**Comment by mjkanji** [comment]
@JackUrb That sounds good to me! Though, I'm not sure why one message use print and the other `click.echo`. I don't have much experience with click/rich_click so if there's a very obvious reason I'm missing, let me know!

```
print("\n[red]Something went wrong.[/red]")
click.echo(e)
```
From my cursory look at `rich-click`, it seems that it doesn't give you the flexibility to route to stderr so would you be okay with changing to a `rich.print` call and then use `file=sys.stderr` as you suggested to route the message correctly?

**Comment by JackUrb** [comment]
@pringshia do you have any intuition as to why some of the cli uses `click.echo` and why some uses `rich.print`? I imagine the prior `echo` calls are just old and all can be updated to `rich.print` for consistency.

**Comment by pringshia** [comment]
Yes, `rich` support came later, so not all of the codebase was transitioned to it. Feel free to change the `click.echo` references to `rich.print`
